index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="theme-color" content="#6070FF" />
+    <link rel="stylesheet" href="css/build/style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+      
+      
+
+    <title>Kingsley's Portfolio</title>
+</head>
+
+<body class="background-color">
+    <header>
+        <nav class="nav-bar">
+            <ul>
+                <li>
+                    <a href=""><div class="logo-font">Kingsfolio</div></a>
+                </li>
+                <li>
+                    <a href=""><i class="fa fa-lg fa-bars menu-icon"></i></a>
+                </li>
+            </ul>
+        </nav>
+
+        <div class="banner">
+            <div class="banner-header banner-content">
+                I'm Kingsley, Glad to see you!
+            </div>
+
+            <div class="body-text-2 banner-content">
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
```

Comment:
- [ ] Also this one it's a paragraph so it would be better if you use `<p>` instead of `<div>` in order to improve the SEO of your portfolio 
- please try always to use the right semantic tags as much as possible  instead of `<div>`